src/config/kernel.md

```diff
@@ -51,6 +51,55 @@ that is currently booted:
 # vkpurge rm all
 ```
 
+## Microcode
+
+Microcode is loaded onto the CPU or GPU at boot, by the BIOS, but can be
+replaced later in the boot stage by the OS itself. An update to microcode can
+allow a CPUs or GPUs low level behavior to be modified to work around certain
+yet to be discovered bugs, without needing to replace the hardware.
+
+### Installation
+
+#### Intel
+
+The Intel microcode package is in the non-free repo, so that has to be enabled
+first.
+
+```
+# xbps-install -S void-repo-nonfree
+```
+
+Install the `intel-ucode` package
+
+```
+# xbps-install -S intel-ucode
+```
+
+Then reconfigure the linux kernel. This only has to be done once, the first time
```

Comment:
The intend is to rebuild the initramfs matching the given kernel, you could mention that so that it's clear all existing initramfs files (for other installed kernel packages) need to be rebuilt in order to properly load the microcode